### PR TITLE
Vector index page clean up

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-962.surge.sh

Updated pages:
- [Vector indexes](https://neo4j-docs-cypher-962.surge.sh/cypher-manual/5/indexes/semantic-indexes/vector-indexes)

File: modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc [hunk 1]
```diff
@@ -74,6 +74,7 @@ As of Neo4j 5.16, the index name can also be given as a parameter: `CREATE VECTO
 
 [NOTE]
 The index name must be unique among both indexes and constraints.
```
Comment (stefano-ottolenghi) [inline modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc]:
```suggestion
The index name must be unique among both indexes and constraints. +
```

**Review by stefano-ottolenghi — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-962.surge.sh

Updated pages:
- [Vector indexes](https://neo4j-docs-cypher-962.surge.sh/cypher-manual/5/indexes/semantic-indexes/vector-indexes)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**